utils/dev-tool/src/server.rs

```diff
@@ -0,0 +1,86 @@
+use crate::utils::HashInfo;
+use crate::{panic_if_unsuccessful, utils, ToolEnvironment};
+
+use std::fs;
+use std::process::{Command, Stdio};
+
+pub fn build_server(tool_env: ToolEnvironment) {
+    dotenv::from_path(utils::local_env_path(&tool_env.root_dir)).unwrap();
+
+    let build_results = Command::new("cargo")
+        .args(["build", "-p", "lockbook-server"])
+        .status()
+        .unwrap();
```

Comment:
you need to ask cargo to rename the server otherwise 2 prs will step on each other. Create then rename is not atomic

Reply:
Renaming a binary on build is an unstable feature. I feel like it doesn't matter that much because any rebuild would take more time than rename wouldn't it?

Reply:
then you should probably do `cargo run` instead